### ISSUE TITLE
write a playbook that creates the user

### ISSUE THREAD

**Issue by PC-Admin** [application-research/estuary-hosted-infrastructure#4]
Wings:

> Also, it could be helpful to write a playbook that creates the user:
> 
> "estadmin"
> UID 1336
> GID 1336
> Non system user
> Shell /bin/bash
> 
> And populates it with pubkeys defined in Git.
> 
> That way we can roll out keys to every machine so that we can open up access to boxes to Lodge, Alvin, etc.
> 
> estadmin will be the primary way people log onto boxes until we get LDAP in place.

I'm thinking this could be a part of like a 'standard operating environment' playbook that also resolves issues like this one: https://github.com/application-research/estuary-hosted-infrastructure/issues/1

**Comment by Zorlin** [comment]
Yeah, basically. I was going to call it "base-playbook" in keeping with traditions at previous employers. It's the base of any machine.
